drivers/segger/config/Global.h

```diff
@@ -0,0 +1,45 @@
+/****************************************************************************
+ * drivers/segger/config/Global.h
```

Comment:
I think it'd better to lower case the file name (Global.h -> global.h) as almost all NuttX file names are lower case.

Reply:
drivers/segger/config dir is not use for nuttx, this is overwrite include path for system view:

https://github.com/SEGGERMicro/SystemView/blob/master/SEGGER/SEGGER.h#L59
https://github.com/SEGGERMicro/SystemView/blob/master/Config/Global.h